Tests/PluginGeneratorTests/PluginBuilderTests.cs

```diff
@@ -101,7 +101,7 @@ public void PluginBuilder_Extensions_MultipleValid()
 @"package myPackage1;
 public class MyClass2{}");
             builder.AddSourceFile(secondValidSource);
-            builder.AddExtension("myPackage1.MyClass2");
```

Comment:
Previously the generated code assumed that the it was being supplied a class, so it automatically appended ".class" if it didn't exist. This is no longer the case, so the test needs to explicitly specify that the extension is a class.